auth/src/test/java/com/firebase/ui/auth/testhelpers/TestHelper.java

```diff
@@ -54,28 +54,31 @@
     private static final String DEFAULT_APP_NAME = "[DEFAULT]";
 
     public static final FirebaseApp MOCK_APP;
+    public static Context context = ApplicationProvider.getApplicationContext();
```

Comment:
nit: `CONTEXT`. Also, make private? Otherwise it seems like our others tests should use it (I don't think they should).